projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -1,7 +1,7 @@
 import { CommonModule } from '@angular/common';
 import {
     AfterViewInit, ChangeDetectorRef, Component, ElementRef, NgModule, OnInit, OnDestroy,
-    Optional, Inject, Injector, ViewChild, Input, Output, EventEmitter, HostListener
+    Optional, Inject, Injector, ViewChild, Input, Output, EventEmitter, HostListener, AfterContentChecked
```

Comment:
it is not implementing now `AfterContentChecked`

Reply:
actually, in the end there are no any changes in this component, but some refactoring!

Reply:
The new changes would be the addition of `prefix` & `suffix` props, that's about it.